index.Rmd

```diff
@@ -17,23 +17,18 @@ description: "Documentation for IDFG PopR"
 
 # Introduction
 
-This manual is intended to provide support and guidance for using the 
-[IDFG PopR Website][PopR] to run population models. Models covered here include 
-[Sightability](#sight), [Survival](#surv), and [IPM](#ipm).
+This manual is a support document for the IDFG PopR website, which was created to streamline the process of running population models including [IPM](#ipm), [Sightability](#sight), and [Survival](#surv), as well as a framework for uploading survey [data](#de).
 
-This manual was created by [Speedgoat][spdgt] and [IDFG](https://idfg.idaho.gov/) using [bookdown](https://bookdown.org/).
+At the top right of your page is a help toolbar. From left to right, the question mark brings you to the Speedgoat homepage, the little bug icon allows you to send a bug report to the speedgoat team, the lightbulb icon allows you to send the team an idea you have for a new feature, the broadcast symbol is a link to the speedgoat blog, and the lifering gives direct contact information for Josh and Paul.
```

Comment:
It might help to have a picture of the page with the toolbar you are referring to. This will help with connecting your descriptions to the page itself.